### ISSUE TITLE
onChange not referencing correct component state

### ISSUE THREAD

**Issue by mintyfloss** [dozoisch/react-google-recaptcha#153]
react-google-recaptcha version: 1.1.0
react-async-script version: 1.0.0

In the `onChange` callback, I want to reference the current state of my component. However, I notice that I'm always getting back the initial state instead. Here's a simple example.

```js
const Test = () => {
  const [value, setValue] = useState("initial value");
  const recaptchaRef = useRef();

  return (
    <>
      <input
        type="text"
        value={value}
        onChange={ev => {
          setValue(ev.target.value);
        }}
      />
      <ReCAPTCHA
        ref={recaptchaRef}
        sitekey={recaptchaSiteKey}
        onChange={() => {
          console.log(value); // always returns "initial value"
        }}
      />
    </>
  );
};
```

I type something into the input field to change the state, and then I click on the recaptcha. I'm expecting the console to print out the current value of the input field, but I'm getting `"initial value"` instead.

Am I doing something wrong here?

**Comment by dozoisch** [comment]
So the function is passed to the captcha render directly which only happens at mount time. 

I guess the fix for this would be to have https://github.com/dozoisch/react-google-recaptcha/blob/master/src/recaptcha.js have it's own handleChange 

and in handleChange call this.props.onChange which would then contain the right value. I might have time to do this over the weekend, but if you have time before, I'm open to reviewing a PR

**Comment by mintyfloss** [comment]
@dozoisch You're right. Adding a `handleChange` method fixes this. I'll see if I have time to create a PR.

**Comment by dozoisch** [comment]
released in 2.0.1!
